src/BibleWell.App/Assets/AppResources.axaml

```diff
@@ -19,6 +19,21 @@
     </ResourceDictionary.ThemeDictionaries>
     -->
 
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="IconActiveBrush">DodgerBlue</SolidColorBrush>
```

Comment:
This looks great!  I did some searching and it looks like we can accidentally override the Avalonia Fluent Theme application resources if we happen to use on with the same key string (or maybe they override us?).  Therefore, we should "namespace" our keys.  I'm thinking that the prefix `BW_` should do it (prefixed to all of these application resource keys) but I'm open to other suggestions.